src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx

```diff
@@ -828,8 +835,8 @@ export class Flyout extends Component<FlyoutProps, FlyoutState> {
             notifications={this.props.notifications.toasts}
             onSelectedDataSource={this.onSelectedDataSourceChange}
             disabled={!this.props.dataSourceEnabled}
-            hideLocalCluster={this.props.hideLocalCluster}
             fullWidth={true}
+            isClearable={false}
```

Comment:
Hi Zhongnan, what is this isClearable do? I did not find it in the doc here. https://quip-amazon.com/UepCAPYFGtKC/Use-Data-Source-Menu-to-Nav-Bar-or-Data-Source-Selector-inside-Page#temp:C:eCC8e412f6d533d4057af4524ca8

Reply:
it was a new prop added by this PR https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6298/files, it's being passed to `EuiComboBox`
basically it make sure you can't clear the input, but you can switch to other option.

Reply:
Got it.